ansible/roles/pg_backup/templates/plain/aws_credentials.j2

```diff
@@ -0,0 +1,6 @@
+
+# {{ ansible_managed }}
+
+[default]
+aws_access_key_id={{ localsettings.AMAZON_S3_ACCESS_KEY }}
+aws_secret_access_key={{ localsettings.AMAZON_S3_SECRET_KEY }}
```

Comment:
Is this (and other similar files) just moving from the `postgresql` role? If so I think you forgot to remove the other files.